.gitignore

```diff
@@ -27,4 +29,7 @@ debug.log
 ui-debug.log
 
 # vscode
-/.vscode
\ No newline at end of file
+/.vscode
+
+# env files
+server/.env
```

Comment:
I'm reluctant to commit different versions of .env because dotenv (https://www.npmjs.com/package/dotenv#:~:text=MULTILINE%3A%20%27new%0Aline%27%7D-,FAQ,-Should%20I%20commit) doesn't recommend it but I included a .env.example file with placeholders.